src/uct/ugni/smsg/ugni_smsg_ep.c

```diff
@@ -53,7 +53,7 @@ static ucs_status_t uct_ugni_smsg_mbox_reg(uct_ugni_smsg_iface_t *iface, uct_ugn
     }
     pthread_mutex_lock(&uct_ugni_global_lock);
 
-    ugni_rc = GNI_MemRegister(iface->super.nic_handle, (uint64_t)address,
+    ugni_rc = GNI_MemRegister(uct_ugni_iface_nic_handle(&iface->super), (uint64_t)address,
```

Comment:
It's most useful in UDT, where changes to the structure (like when I was deciding where that field should land in the final structure layout) required changes in a lot of place. This way, change it once and everything compiles fine. I can condense the name. :)